src/Form/UnregistrationType.php

```diff
@@ -18,23 +17,19 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'filter_emojis' => true,
             ])
-            ->add('reasons', UnregistrationReasonsChoiceType::class)
+            ->add('reasons', ChoiceType::class, [
+                'choices' => $options['reasons'],
+                'choice_translation_domain' => 'forms',
+                'expanded' => true,
+                'multiple' => true,
+            ])
         ;
     }
 
     public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults([
-            'data_class' => UnregistrationCommand::class,
-            'validation_groups' => function (FormInterface $form) {
-                $data = $form->getData();
-
-                if (in_array('autre', $data->getReasons())) {
-                    return ['Default', 'Reason'];
-                }
-
-                return ['Default'];
-            },
+            'reasons' => [],
```

Comment:
need to keep the `data_class` option, no?

Reply:
of course! thx :+1: